### PR TITLE
[3829][15.0][UPD] dotfiles

### PR DESCRIPTION
**Pull request by liuhehe1995**
[3829](https://www.quartile.co/web#id=3829&menu_id=505&cids=3&action=1457&model=project.task&view_type=form)

### CODE REVIEW

File: .copier-answers.yml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # Do NOT update manually; changes here will be overwritten by Copier
-_commit: v1.14.2
+_commit: v1.19.1
 _src_path: https://github.com/OCA/oca-addons-repo-template.git
 ci: GitHub
-dependency_installation_mode: PIP
+convert_readme_fragments_to_markdown: false
```
Comment (AungKoKoLin1997) [inline .copier-answers.yml]: This is your answer or default?

Comment (liuhehe1995) [reply .copier-answers.yml]: It is  default.
I commit again.

**Review by AungKoKoLin1997 — APPROVED** [review]
@liuhehe1995 Please squash the commits.

**Review by kanda999 — APPROVED** [review]
LGTM

**Pull request merged**